aloe_webdriver/util.py

```diff
@@ -508,3 +510,35 @@ def wrapped(*args, **kwargs):
                     raise
 
     return wrapped
+
+
+RETRIES = 5
+
+
+def retry_on_timeout(func):
+    """
+    A decorator to invoke a function, retrying on timeout exceptions for a
+    specified number of times.
+
+    Adds a kwarg `retires` to `func` which is the number of times to try
+    for (default 5).
+    """
+
+    def wrapped(*args, **kwargs):
```

Comment:
You can add the functools.wraps decorator to this function to have the convenience of copying the function name, docstring, etc. to the functions you decorate with `@retry_on_timeout`.

https://docs.python.org/2/library/functools.html#functools.wraps